source/template.yaml

```diff
@@ -49,6 +49,12 @@ Globals:
         NO_PROXY: !Sub
           - '{{resolve:ssm:/${Environment}/network/outboundproxy/no_proxy:1}}'
           - Environment: !Ref environment
+        ADDRESS_PROOF_RESULT_LAMBDA_TOKEN: !Sub
+          - '{{resolve:ssm-secure:/${Environment}/idp/doc-capture/address_proof_result_token:1}}'
```

Comment:
what's the `:1` for?

Reply:
Are these applied to all functions? Would it make sense to only share the address proof secret with `address_proof` and `address_proof_mock` ?

Reply:
The :1 is for the version number of the parameter.

I moved the environment variables from the global section to the respective functions.